### PR TITLE
FAT-1296: Implement e-2-e automation of test case C347828

### PR DESCRIPTION
**Pull request by mariia-aloshyna**
https://foliotest.testrail.io/index.php?/cases/view/347828

### CODE REVIEW

File: cypress/integration/data-import/match-on-instance-identifier-scen1.spec.js [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+    typeValue : NewActionProfile.folioRecordTypeValue.instance,
+    action: 'Update (all record types except Orders, Invoices, or MARC Holdings)'
+  };
+  const jobProfile = {
+    profileName: jobProfileName,
+    acceptedType: NewJobProfile.acceptedDataType.marc
+  };
+
+  before(() => {
+    cy.createTempUser([
```
Comment (penfriend) [inline cypress/integration/data-import/match-on-instance-identifier-scen1.spec.js:71]: Here you are using cy.getAdminToken function. There is no need to call this function in the line 85. Please move the block of code from 87 line to 82 line.
Any time you call login function(cy.getAdminToken, cy.login) you reset the previous login session.

File: cypress/integration/data-import/match-on-instance-identifier-scen1.spec.js [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+      permissions.viewEditDeleteInvoiceInvoiceLine.gui,
+      permissions.viewEditCreateInvoiceInvoiceLine.gui,
+      permissions.assignAcqUnitsToNewInvoice.gui,
+      permissions.invoiceSettingsAll.gui,
+    ])
+      .then(userProperties => {
+        userId = userProperties.userId;
+        cy.login(userProperties.username, userProperties.password);
+      });
+    cy.getAdminToken();
```
Comment (penfriend) [inline cypress/integration/data-import/match-on-instance-identifier-scen1.spec.js:85]: If you move InventorySearch.getInstancesByIdentifierViaApi function to the 82 line. The calling of this function would be obsolete

File: cypress/integration/data-import/match-on-instance-identifier-scen1.spec.js (hunk 1 above)
Comment (penfriend) [reply cypress/integration/data-import/match-on-instance-identifier-scen1.spec.js:71]: Actually by calling on line 85  the function cy.getAdminToken(); you reset the session of login the user from the line 83

File: cypress/integration/data-import/match-on-instance-identifier-scen1.spec.js [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+    InventorySearch.getInstancesByIdentifierViaApi(resourceIdentifiers[0].value)
+      .then(instances => {
+        instances.forEach(({ id }) => {
+          InventoryInstance.deleteInstanceViaApi(id);
+        });
+      });
+  });
+
+  after(() => {
+    Users.deleteViaApi(userId);
```
Comment (TetianaParanich) [inline cypress/integration/data-import/match-on-instance-identifier-scen1.spec.js:96]: please add deleting created profiles

**Review by TetianaParanich — APPROVED** [review]

**Comment by penfriend** [comment]
@mariia-aloshyna  we usually merge our pull requests with minimum two approvals. Please wait until we have a time to review a pull request at first or second time.

**Pull request merged**